### PR TITLE
Remove redundant migration and add needed migration

### PR DESCRIPTION
**Pull request by Tomanhez**
Migration Version20200301170604 duplicated logic from Version20200202104152, Version20200122082429, Version20200110132702

### CODE REVIEW

**Comment by Zales0123** [comment]
Fixes https://github.com/Sylius/Sylius-Standard/issues/399, I suppose

File: src/Migrations/Version20200512134107.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_channel DROP type');
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_channel ADD type VARCHAR(255) CHARACTER SET utf8 DEFAULT NULL COLLATE `utf8_unicode_ci`');
```
Comment (lchrusciel) [inline src/Migrations/Version20200512134107.php:23]: For sure this shouldn't land in the master. We don't have this field at all. 

Also, we should never create new migrations for standard I suppose, rather move ones from the main Sylius repository.

**Comment by lchrusciel** [comment]
Fixes #395 as well

**Comment by lchrusciel** [comment]
Thanks, Tomasz! :1st_place_medal:

**Pull request merged**